### DISCUSSION TITLE
Exports turned off after stockpiling for a request

### DISCUSSION THREAD

**Discussion by blubb975** [Keriew/augustus#466] [Ideas]
New here so if I do any formatting issues please tell me

Let's say I have a good set to export. Then Caesar does a request for that good, I have less of it in store than what is being demanded, so I click the "store" button to temporarily disable use and sales of that good. It stocks up, I send it off, making sure to not tick the "continue stockpiling" box. Now it isn't re-set to exporting again so I have to manually go to the trade advisor to re-enable exports.
Also happens when I click on "store" and then "stop storing" - the export stays turned off.

Using the 3.0.1 stable

Here a save with a weapons request:  [save](https://drive.google.com/file/d/1Qe-aO_Q-7Ak_Lj9Cqjpt6I9nl6hbc5U7/view?usp=sharing)  [ini](https://drive.google.com/file/d/1gitGybcvy8BdYpnl5vzFKFcHVhQOxfuf/view?usp=sharing)

**Comment by crudelios** [comment]
This is the default behaviour of the game and we left it like this by design - you may not want to keep exporting the stockpiled good after you're done stockpiling it.

We may change it though if it makes more sense.

**Comment by blubb975** [comment]
Ah I see, thanks for the explanation. In vanilla C3 though you have to go to the trade advisor anyway if you want to start/stop stockpiling stuff, so in that case you instantly see that it's no longer exporting. In contrast to that, in Augustus this behaviour makes the export stop easier to overlook when using the new right hand panel in city view. Would definitely be nice to have it as an option to automatically export again

**Comment by Riww** [comment]
Agree with blubb, the export stop is not intuitive. Also, for uniformity, I think it should always be called "stockpile" instead of "store", as it is the same thing.
